source/_integrations/home_connect.markdown

```diff
@@ -41,7 +41,7 @@ Note that it depends on the appliance and on API permissions which of the featur
 
 - Application ID: Home Assistant (or whatever name makes sense to you)
 - OAuth Flow: Authorization Code Grant Flow
-- Redirect URI: `https://my.home-assistant.io/redirect/oauth`
+- Redirect URI: `https://my.home-assistant.io/auth/external/callback`
```

Comment:
This change is incorrect.
The provided URL is 404 as well.

The link in the existing documentation is the correct one.